framework/base/Controller.php

```diff
@@ -67,6 +66,10 @@ class Controller extends Component implements ViewContextInterface
      * @var View the view object that can be used to render views or view files.
      */
     private $_view;
+    /**
+     * @var string the root directory that contains view files for this controller.
+     */
+    private $_viewPath;
```

Comment:
`@since` is not really useful on private properties as they do not belong to the public api, it does not matter when they were introduced.